Common/src/CConfig.cpp

```diff
@@ -1150,6 +1150,8 @@ void CConfig::SetConfig_Options() {
   /*!\par CONFIG_CATEGORY: FluidModel \ingroup Config*/
   /*!\brief FLUID_MODEL \n DESCRIPTION: Fluid model \n OPTIONS: See \link FluidModel_Map \endlink \n DEFAULT: STANDARD_AIR \ingroup Config*/
   addEnumOption("FLUID_MODEL", Kind_FluidModel, FluidModel_Map, STANDARD_AIR);
+  /*!\brief FLUID_NAME \n DESCRIPTION: Fluid name \n OPTIONS: see coolprop homepage \n DEFAULT: nitrogen \ingroup Config*/
+  addStringOption("FLUID_NAME", FluidName, string("nitrogen"));
 
 
```

Comment:
In `CConfig::SetPostprocessing` add an error if CoolProp is used with non-dimensionalization.

Reply:
I will do this as soon.

Reply:
Hi, I meet a problem when I use git push origin feature_coolprop.

The error message is **'permission to su2code/SU2.git denied'**

should I merge to develop branch from my fork, rather than from SU2/feature_coolprop?

If so, should I open a PR and close this PR?

Thank you.

Reply:
you should checkout SU2/feature_coolprop, make changes to that branch locally (including merging one of your other branches into feature_coolprop), and then do only git push.
If you are having to do git push origin you probably created a new branch instead of modifying this one.
Be careful not do to any force pushes or anything "irreversible"

Reply:
Thank you for your help.

I am sorry that I meet the same problem. Let me describe what I did in details.

1. I download SU2 by running $git clone https://github.com/su2code/SU2.git  --branch=feature_coolprop
2.  use meson and ninja to build SU2
3. modify SU2 source code
4. git add .
5. git commit -m ''
6. git push origin feature_coolprop
Then git told me permission denied.
7. I tried to use 'git push'
Permission denied again.

So what should I do now? should I remove SU2 and install and build it again from scratch, then modify the source code, finally use git push.

Thank you.

Reply:
See if anything here helps:
https://su2code.github.io/docs_v7/Developing-SU2-on-GitHub-(Internal-Developers)/

If it doesnt, push to your fork and then create a PR from a branch in your for to this branch in SU2. Do not close this PR.

Reply:
The online document does not help for me.

I will open a new PR and merge from my fork to SU2/feature_coolprop.

Thank you.